src/main/java/profiler/generation/R1CSConstruction.java

```diff
@@ -358,6 +353,48 @@
         return new Tuple3<>(r1cs, primary, oneFullAssignment);
     }
 
+    public static <FieldT extends AbstractFieldElementExpanded<FieldT>>
+        Tuple3<R1CSRelation<FieldT>, Assignment<FieldT>, Assignment<FieldT>>
+    serialSmallestExample(final FieldT fieldFactory) {
```

Comment:
Is this function used anywhere? What's its purpose?

Reply:
It only exists at the moment because it was used early on as a test case. I think it kinda serves the purpose of ensuring the smallest possible case is always satisfied (considering throughout this entire project they only use big examples). One wouldn't want the entire universe to implode on itself by neglecting the trivialities. Perhaps I can include a unit test that makes use of this (so that it is used).

Reply:
Has been removed.